lib/tools/system-calls.js

```diff
@@ -294,11 +294,11 @@ systemCallMethods.adbExec = async function (cmd, opts = {}) {
       }
       let args = this.executable.defaultArgs.concat(cmd);
       log.debug(`Running '${this.executable.path} ${quote(args)}'`);
-      let {stdout} = await exec(this.executable.path, args, opts);
+      let {stdout, stderr} = await exec(this.executable.path, args, opts);
```

Comment:
I thought that stderr is only present if error code is not equal to zero, but whatever

Reply:
i would also be fine parameterizing this method to optionally return stderr